### PR TITLE
DAC - Non-text Contrast (AA)

### PR DESCRIPTION
**Pull request by marianayovcheva**
https://citizensadvice.atlassian.net/browse/NP-2777
https://citizensadvice.atlassian.net/browse/NP-2778

Design: https://www.figma.com/file/Wecjoy7kvEs6ZCqWUBnxlH/Design-System%3A-Work-in-progress?node-id=496%3A16934
Changing the focus states of the input fields and targeted content. The date field currently exists in public-website only but looking at the code it should inherit the needed design system input styles. I will test once we upgrade the design system version on public-website.

### CODE REVIEW

File: scss/2-tools/_interactive-elements.scss [hunk 1]
```diff
@@ -1,8 +1,9 @@
 @mixin cads-interactive-focus() {
-  outline: none;
-  border-color: $cads-language__focus-border-colour;
+  outline: $cads-border-width-medium solid $cads-language__focus-colour;
```
Comment (marianayovcheva) [inline scss/2-tools/_interactive-elements.scss:2]: I had to add outline as the input value was moving slightly when the user focuses on the field. I used the gov.uk approach

**Comment by davidrapson** [comment]
This looks great. Noticed whilst testing that the search form both in the header and standalone still have the previous focus styles. So we probably need to apply the same styles there, but generally looks great so far.

File: scss/6-components/forms/form-groups.scss [hunk 2]
```diff
@@ -82,7 +82,8 @@ $cads-checkable-size: 40px;
 
 .cads-form-group__input:focus + .cads-form-group__label {
   &::before {
-    border-color: $cads-language__focus-border-colour;
+    border-color: $cads-language__focus-text-colour;
+    border-width: 3px;
```
[the remark this answers is not in the document]
Comment (davidrapson) [reply scss/6-components/forms/form-groups.scss:86]: I don't think it needs to be a top-level variable but maybe having it as a `$cads-checkable-` prefixed variable in this file like we already do for `$cads-checkable-size`?

**Comment by marianayovcheva** [comment]
Good spot, thanks @davidrapson! I've now flagged with @JamesR-CA and we will work on tweaks for the search form as well. Probably a candidate for a different PR

**Review by davidrapson — APPROVED** [review]
👍 Sounds good, happy to approve this one as is then!

**Pull request merged**